packages/sdk/src/RequestController.ts

```diff
@@ -53,7 +53,13 @@ export class RequestController {
 		return response.data
 	}
 	public async post<T>(path: string, body: Record<string, any>, params?: Record<string, any>): Promise<T> {
-		const response = await this.request.post<T>(path, body, { params })
+		let response
+		if (params?.enableDoubleOptIn) {
```

Comment:
Low level entity such as `RequestController` should not care about whatever something as `enableDoubleOptIn` is enabled and do different logic based on that. It would be better just to add 4 param, `headers?: <Record, string, string>` and pass it to `.post`